src/conventional/metar_csv2ioda.py

```diff
@@ -48,6 +93,23 @@ def __init__(self, filename, date):
 
     def _rd_metars(self):
 
+        n = 0
+        for iodavar in obsvars.values():
+            self.varDict[iodavar]['valKey'] = iodavar, iconv.OvalName()
+            self.varDict[iodavar]['errKey'] = iodavar, iconv.OerrName()
+            self.varDict[iodavar]['qcKey'] = iodavar, iconv.OqcName()
+            self.varAttrs[iodavar, iconv.OvalName()]['coordinates'] = 'longitude latitude'
+            self.varAttrs[iodavar, iconv.OerrName()]['coordinates'] = 'longitude latitude'
+            self.varAttrs[iodavar, iconv.OqcName()]['coordinates'] = 'longitude latitude'
+            self.varAttrs[iodavar, iconv.OvalName()]['units'] = obsvars_units[n]
+            self.varAttrs[iodavar, iconv.OerrName()]['units'] = obsvars_units[n]
```

Comment:
Please add units for the iconv.OqcName version, using 'unitless' for the units value. Thanks!

Reply:
Thanks for taking care of this.